layout/_macro/post.swig

```diff
@@ -274,7 +270,7 @@
                   <i class="fa fa-clock-o"></i>
                 </span>
                 {% if theme.symbols_count_time.item_text_post %}
-                  <span class="post-meta-item-text">{{ __('symbols_count_time.time') }} &asymp;</span>
+                  <span class="post-meta-item-text">{{ __('symbols_count_time.time') + __('symbol.colon') }}</span>
```

Comment:
I think no need to touch `&asymp` (≈) symbol. It's pretty look and show to user rounding time to read.

Reply:
yeah, i forget it's ≈ symbol, it will be reverted.

The changes are deployed in http://test.saili.science/

1.  nominalize the post meta of `modified time` and `post folder` for convenient translation..
2.  change the logic mentioned in [comments](https://github.com/theme-next/hexo-theme-next/pull/228#issuecomment-389430077).